### PR TITLE
Remove useless else

### PR DESCRIPTION
**Pull request by lucasmichot**
Remove useless else

### CODE REVIEW

File: src/Commands/JWTGenerateSecretCommand.php [hunk 1]
```diff
@@ -59,9 +59,9 @@ public function fire()
                     file_put_contents($path, str_replace(
                         'JWT_SECRET='.$this->laravel['config']['jwt.secret'], 'JWT_SECRET='.$key, file_get_contents($path)
                     ));
-                } else {
-                    return $this->comment('Phew... No changes were made to your secret key.');
                 }
+
+                return $this->comment('Phew... No changes were made to your secret key.');
```
Comment (tymondesigns) [inline src/Commands/JWTGenerateSecretCommand.php]: This one is necessary. If you answer "yes" then it will still output as if no changes were made

Comment (tymondesigns) [inline src/Commands/JWTGenerateSecretCommand.php]: Guess I should take another look at that command test...

Comment (lucasmichot) [inline src/Commands/JWTGenerateSecretCommand.php]: My bad @tymondesigns
See: 600866c73448ad12b547f210d75fa7f79a68c655

**Comment by tymondesigns** [comment]
:+1:

**Pull request merged**